.env.example

```
@@ -1,3 +1,4 @@
- ELEVENLABS_API_KEY=PUT_YOUR_KEY_HERE
- ELEVENLABS_MCP_BASE_PATH=~/Desktop # optional base path for output files
- ELEVENLABS_API_RESIDENCY="us" # optional data residency location
\ No newline at end of file
+ELEVENLABS_API_KEY=PUT_YOUR_KEY_HERE
+ELEVENLABS_MCP_BASE_PATH=~/Desktop # optional base path for output files
+ ELEVENLABS_API_RESIDENCY="us" # optional data residency location
```

dotenv-linter (3.3.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)